.github/workflows/e2e-tests.yml

```diff
@@ -47,6 +47,10 @@ jobs:
       # This uses GitHub Action cache for docker container layers
       # See: https://github.com/marketplace/actions/docker-layer-caching
       - uses: satackey/action-docker-layer-caching@v0.0.11
+        # Disable for now, to avoid storage space error.
+        # #TODO investigate how to run this without error.
```

Comment:
Is there an issue on the board for this todo?

Reply:
Ended up removing this section, as it seems clear enough that this isn't helping us :)